<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefNetwork</class>
 <widget class="QWidget" name="PrefNetwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>514</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="youtube_page">
      <attribute name="title">
       <string>&amp;YouTube (and other sites)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Support for &amp;video sites:</string>
           </property>
           <property name="buddy">
            <cstring>streaming_type_combo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="streaming_type_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="youtube_box">
         <property name="title">
          <string>Options for YouTube</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="_3">
            <item>
             <widget class="QLabel" name="yt_label">
              <property name="text">
               <string>Playback &amp;quality</string>
              </property>
              <property name="buddy">
               <cstring>yt_quality_combo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="yt_quality_combo"/>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="yt_user_agent_label">
              <property name="text">
               <string>&amp;User agent</string>
              </property>
              <property name="buddy">
               <cstring>yt_user_agent_edit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yt_user_agent_edit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cc_page">
      <attribute name="title">
       <string>C&amp;hromecast</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Web Server</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="6" column="0" colspan="3">
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Changes in this section will be applied the next time the web server is restarted</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="port_spin">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QCheckBox" name="directory_listing_check">
            <property name="text">
             <string>&amp;Directory listing</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Local &amp;IP:</string>
            </property>
            <property name="buddy">
             <cstring>local_ip_combo</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>179</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>&amp;Port:</string>
            </property>
            <property name="buddy">
             <cstring>port_spin</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>179</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>In order to serve local media from this computer to Chromecast, SMPlayer will run a tiny web server. You can adjust here some of the settings.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="local_ip_combo">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="subs_box">
         <property name="title">
          <string>Subtitles</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="convert_subs_check">
            <property name="text">
             <string>Convert SRT subtitles to &amp;VTT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="subtitle_container" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="margin">
              <number>0</number>
             </property>
             <item row="0" column="0" rowspan="2">
              <spacer>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>21</width>
                 <height>61</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="overwrite_subs_check">
               <property name="text">
                <string>&amp;Overwrite existing VTT files</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="subs_filter_check">
               <property name="text">
                <string>Try to &amp;remove advertisements</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Position of &amp;subtitles on screen:</string>
                 </property>
                 <property name="buddy">
                  <cstring>subs_position_spin</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="subs_position_spin">
                 <property name="minimum">
                  <number>-1</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="proxy_page">
      <attribute name="title">
       <string>&amp;Proxy</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="use_proxy_check">
         <property name="text">
          <string>&amp;Enable proxy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="proxy_container" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Host:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_hostname_edit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="proxy_hostname_edit"/>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&amp;Port:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>proxy_port_spin</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QSpinBox" name="proxy_port_spin">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&amp;Username:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_username_edit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="4">
           <widget class="QLineEdit" name="proxy_username_edit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Pa&amp;ssword:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_password_edit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QLineEdit" name="proxy_password_edit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&amp;Type:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_type_combo</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="proxy_type_combo"/>
          </item>
          <item row="3" column="2" colspan="3">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>261</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tab_widget</tabstop>
  <tabstop>streaming_type_combo</tabstop>
  <tabstop>yt_quality_combo</tabstop>
  <tabstop>yt_user_agent_edit</tabstop>
  <tabstop>local_ip_combo</tabstop>
  <tabstop>port_spin</tabstop>
  <tabstop>directory_listing_check</tabstop>
  <tabstop>convert_subs_check</tabstop>
  <tabstop>overwrite_subs_check</tabstop>
  <tabstop>subs_filter_check</tabstop>
  <tabstop>subs_position_spin</tabstop>
  <tabstop>use_proxy_check</tabstop>
  <tabstop>proxy_hostname_edit</tabstop>
  <tabstop>proxy_port_spin</tabstop>
  <tabstop>proxy_username_edit</tabstop>
  <tabstop>proxy_password_edit</tabstop>
  <tabstop>proxy_type_combo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>use_proxy_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxy_container</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>convert_subs_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>subtitle_container</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
